### ISSUE TITLE
The certificate of ‘himawari8.nict.go.jp’ is not trusted

### ISSUE THREAD

**Issue by adriel** [domoritz/himawari-8-chrome#186]
Anyone else getting no image for "Himawari Visable Light"?

Looking at the console looks like there are cert errors. Even trying to pull down a tile on a different connection still get the cert errors. It's been like this since maybe a week or 2.

```
$ wget 'https://himawari8.nict.go.jp/himawari8/img/D531106/1d/550/2020/06/26/023000_0_0.png'
--2020-06-26 15:06:02--  https://himawari8.nict.go.jp/himawari8/img/D531106/1d/550/2020/06/26/023000_0_0.png
Resolving himawari8.nict.go.jp (himawari8.nict.go.jp)... 133.243.243.104
Connecting to himawari8.nict.go.jp (himawari8.nict.go.jp)|133.243.243.104|:443... connected.
ERROR: The certificate of ‘himawari8.nict.go.jp’ is not trusted.
ERROR: The certificate of ‘himawari8.nict.go.jp’ doesn't have a known issuer.
```

![image](https://user-images.githubusercontent.com/154217/85816302-0ba1a180-b7bf-11ea-9ca5-f58d6dc80d4f.png)

**Comment by MalTeeez** [comment]
Can confirm, the Himawari option is broken since 3 weeks

**Comment by domoritz** [comment]
Thanks for reporting. I’ll look into it. Hopefully it’s not that their server certificate is not working.

**Comment by adriel** [comment]
I think it might be. But `http` does work. Though of course not ideal.

**Comment by domoritz** [comment]
Fixed in 0.19.1. Thanks for helping me track down the issue @adriel and @MalTeeez.
